packages/lib/src/components/LoadingSpinner/LoadingSpinner.tsx

```diff
@@ -17,17 +17,23 @@
  * along with Essencium Frontend. If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { Loader } from '@mantine/core'
+import { Container, Loader, LoaderProps } from '@mantine/core'
 import { useEffect, useState } from 'react'
 
-interface Props {
+type Props = {
   show: boolean
   delay?: number
+  color?: LoaderProps['color']
+  size?: LoaderProps['size']
+  variant?: LoaderProps['variant']
 }
```

Comment:
I would suggest using it like this because if Mantine updates the component, we automatically support new props:

`type Props = LoaderProps & { show: boolean, delay?: number }`

So instead of having only these props available:

![image](https://github.com/Frachtwerk/essencium-frontend/assets/54100938/d97ff6bd-a4b6-4178-ac62-52856aba0b5f)

We now have these (and the list goes on):

![image](https://github.com/Frachtwerk/essencium-frontend/assets/54100938/8ca4a461-dbc5-4b14-b77a-4290113da648)

And since this is only the typing, we also need to accept the de facto data as props:

![image](https://github.com/Frachtwerk/essencium-frontend/assets/54100938/96e5f6d4-033f-438a-bbf5-3c1008a80895)

And spread them into the core Mantine component

![image](https://github.com/Frachtwerk/essencium-frontend/assets/54100938/5bb390b3-5d81-4018-9a69-2dd09ac1942b)

And do you know whether we can set the default color to the primary theme color? 🤔 

If you need assistance just hit me up 🚀

Reply:
Thanks for the comment, that makes a lot of sense :) I changed it accordingly. 

If we don't define a specific color, the primary theme color is used. The same applies for variants. 
So I removed the default values for color and variant. Then you can either leave it undefined and use the theme settings or set a specific individual value. I kept the default for size though, because we use a custom size here.

Reply:
Perfect 🌈